src/config/config.js

```diff
@@ -177,6 +177,11 @@ export default {
    * @property {Boolean} ignoreMalformedPackets Ignore malformed packets on parsing instead of throwing an error
    */
   ignoreMalformedPackets: false,
+  /**
+   * @memberof module:config
+   * @property {Array} additionalAllowedPackets Allow additional packets on parsing. Defined as array of packet classes, e.g. [PublicKeyPacket]
```

Comment:
I'd add a comment that this config should be set locally, for specific function calls (e.g. `readMessage`). Otherwise it's quite unclear how a global setting is useful IMO.

Reply:
I added a comment to describe the option in more detail. Should we change the initial value in the config from `[]` to `undefined`? If we don't want to have it in the global config anyway and this would simplify the check on initial in the coding.

Reply:
I would leave `[]` so that it's clearer what kind of input is expected. Regarding the check, I commented [below](https://github.com/openpgpjs/openpgpjs/pull/1618/files#r1149186658) but I think it's actually simpler as-is

Reply:
Either way is fine by me, but even now already you can simplify the check I think, to `if (config.additionalAllowedPackets.length)`, since it should always be an array. If you make it undefined by default, then indeed you can do `if (config.additionalAllowedPackets)`, I guess that's even simpler indeed ^.^

Reply:
I changed the check to `if (config.additionalAllowedPackets.length)`. Looks good to me now overall.